packages/rum/test/e2e/async-tests/async.e2e-spec.js

```diff
@@ -50,13 +50,19 @@ describe('async-tests', function () {
      * Check for all types of spans that would be captured by
      * loading script async
      */
-    const spanTypes = ['hard-navigation', 'external']
+    const spanTypes = [
+      'hard-navigation',
+      'external',
+      'longtask',
+      'app',
+      'resource'
+    ]
     /**
-     * Safari does not support Resource & User Timing API
+     * Safari version configured in the test report span type browser-timing
      */
     const { name } = getBrowserInfo()
```

Comment:
Seems like you would need to change this function to return the correct browser name and version with the new updated capabilities?

Reply:
The update to make use of the new capabilities is within that function:

`That function returns the values properly:

`
function getBrowserInfo() {
  const { browserVersion, browserName } = browser.capabilities
  return {
    name: browserName.toLowerCase(),
    version: browserVersion
  }
}
``